### DISCUSSION TITLE
Time of Day Adjustments improvements

### DISCUSSION THREAD

**Discussion by tkorh** [xanderfrangos/twinkle-tray#822] [Ideas]
Hi!

I really like the app, but it seem that the Time of Day Adjustments are a bit wonky. It seems that it does not update the brightness if you turn on the PC after the time you set the brightness to change (for example, I have set it to 05:00 to change back to default brightness, but the monitor stayed at 0%, this is the set value at 22:00, when I turned PC on at around 07:00).

Maybe there is a bug here or I am doing something wrong. In any case, this option is a bit unintuitive and tedious to set up. It would be much better that the user could input points on the graph (x-axis would be time, from 00:00 to 23:59, y-axis would be brightness level). This way user could quickly define the time and brightness levels and also how fast the brightness transitions from one point to another (defined by the slope).

Just an idea.

I did a quick search and I did not find similar suggestion. If there way one, I apologise.

I am currently on Windows 11, v1.15.5.

Keep up the good work!
Cheers!

**Comment by xanderfrangos** [comment]
Hi @tkorh,

Yeah, the ToD feature is a bit buggy in v1.15.5. I believe it gets applied correctly at startup in the v1.16.0 beta. If you want to check out the beta and are using the Microsoft Store version of Twinkle Tray, you'll need to uninstall Twinkle Tray first.

**Reply by tkorh** [reply]
Thank you for a quick reply! I am glad that this is/will be fixed!
